setup.cfg

```diff
@@ -69,7 +69,8 @@ install_requires =
     tinynetrc >= 1.3.1
     toml >= 0.10.0
     mock >= 3.0.5
-
+    # packaging >= 21.3
```

Comment:
We still use packaging in pip-requirements-parser and you cannot remove it. Only add packvers. See in ScanCode toolkit the proper minimal versions to use  here and to pin in the requirements.txt file. You also need to bump dparse2 and pip-requirements-parser here and in requirements.txt

Reply:
Done in https://github.com/nexB/python-inspector/pull/109/commits/d542835d9480e86cd03e493a6f4badd345534721 and https://github.com/nexB/python-inspector/pull/109/commits/06ea29bf33efa7baed2c158205bbcb3cddad50e0